pkg/networkmanager/interfaces.js

```diff
@@ -2347,10 +2347,7 @@ PageNetworkInterface.prototype = {
             $('#networking-firewall').show();
             renderFirewallState();
 
-            var n = firewall.enabledServices.size;
-
-            /* HACK: use n.toString() here until cockpit.format() handles integer 0 args correctly */
-            var summary = cockpit.format(cockpit.ngettext("$0 Active Rule", "$0 Active Rules", n), n.toString());
+            const summary = cockpit.format(_("$0 Active Zones"), firewall.activeZones.size);
```

Comment:
Can it not happen that firewall.activeZones.size is zero? Or did we fix that cockpit.format() bug?

Reply:
when I replaced it with 0 it seemed fine, but I guess I better actually check :p

Reply:
it seems it got handled in #9062

Reply:
Should this use `ngettext`? Now it can be `1 Active Zones`, or `cockpit.format` can deal with it?

Reply:
oh you're right, yea i'l ltag this for tomorrow.

Reply:
Argh, well spotted @marusak!